src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = days * 40;
+  const discountThree = -20;
```

Comment:
Make the discount a positive number and subtract it, so the code will be more readable.